### ISSUE TITLE
[Features] Inline Changelog/Settings Cog

### ISSUE THREAD

**Issue by xlxAciDxlx** [ebiggz/MixrElixr#40]
Similar to how a lot of extensions/plugins work, it'd be nice if two things could be done to increase user awareness:
- *Inline changelog*: essentially a changelog that would either open a new tab or just popup (once) on Mixer displaying what you've changed
- *Settings Cog*: Somewhere on Mixer, it'd be nice to include a cog icon (maybe near XP/sparks?) to access any settings for MixrElixr. I can write up some mockups if you'd like to see some implementation ideas I've had.

Thanks for your time :)

**Comment by ebiggz** [comment]
I like the idea of a changelog popup in mixer whenever we have an update. Definitely will get that added.

At the beginning of development we actively decided against an inline button to access settings for the time being, firstly because it vastly simplifies and improves development speed and quality but also because we knew we wanted to add other things such as the "Who's online" tab that we want available outside of the mixer website. We would like to eventually get a subset of the settings in the site eventually, though, its just not a priority until we've gotten more of the bigger features out of the way.

**Comment by xlxAciDxlx** [comment]
Added another suggestion to my OP, also updated the formatting of the post so you can check things off as they are completed.

I read the contributing guidelines, does this mean we can do things ourselves and submit them directly via Pull Requests? If so I wouldn't mind taking a stab at my own suggestions (at least the ones you don't directly say no to :P).

**Comment by ebiggz** [comment]
Yeah I saw the new favorite button and the new hosting goodies (new host bar, tabbed chat - which Elixr doesn't seem to work 100% on), and was thinking the exact same thing!

Of course, we love seeing the awesome stuff people make for Elixr and Mixer. The inline favorite button was actually done by community member @murfguy, he would be a good resource for that specific feature. :)

**Comment by xlxAciDxlx** [comment]
So I did a bit of a mockup to see if I did go through with it how it'd look, here's what I came up with so far (ideally just to match the "Follow/Following" button.

https://imgur.com/a/j0wMW
